src/main/kotlin/racing/CarRacing.kt

```diff
@@ -13,9 +14,25 @@ object CarRacing {
 
     private fun List<Car>.racePerRound() {
         forEach { car ->
-            car.moveOrStop()
+            car.moveOrStop(random(0..9))
         }
     }
 
-    private fun List<Car>.recordRacingResultPerRound() = map { it.position }
+    private fun random(range: IntRange) = Random.nextInt(range.first, range.last)
+
+    private fun List<Car>.recordRacingResultPerRound() = map { it.copy() }
+
+    private fun findWinners(): List<String> {
+        val maxPosition = cars.maxBy { it.position }.position
+        return cars.filter { car -> car.position == maxPosition }.map { it.name }
```

Comment:
객체지향 생활 체조 9가지 원칙
4. 한 줄에는 점을 하나만 찍는다.
```suggestion
        val maxPosition = cars.maxBy { it.position }
            .position
        return cars.filter { car -> car.position == maxPosition }
            .map { it.name }
```